src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(number) {
+    return function(value) {
+      count++;
+
+      const result = value + number;
+
+      if (count % 2 === 0 && count > 3) {
```

Comment:
You should use one variable for counting function calls. Here, you are using `count` correctly to keep track of the calls to the returned function.